manifests/Adobe/flash-player-ppapi/32.0.0.371.yaml

```diff
@@ -0,0 +1,17 @@
+Id: Adobe.flash-player-ppapi
+Publisher: Adobe
+Name: Adobe Flash Player ppapi
```

Comment:
Please update name to 'Adobe Flash Player 32 PPAPI'
Please also update the Id and the manifest file path accordingly.
For reference see files in the Adobe folder https://github.com/microsoft/winget-pkgs/tree/master/manifests/Adobe
Thanks!